### PR TITLE
Modulesync 4.1.0

### PR DESCRIPTION
**Pull request by root-expert**
#### Pull Request (PR) description

Modulesync and fixes broken acceptance tests by writing a secret to a file and examining it afterwards

#### This Pull Request (PR) fixes the following issues

Closes #33

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 6.0.0"
+      "version_requirement": ">= 6.1.0 < 8.0.0"
```
Comment (smortex) [inline metadata.json:86]: Should be handled by:
https://github.com/voxpupuli/puppet_metadata/commit/491aa993479260ef37b9a926ab4b3aa07c1e3fba

The requirement for puppet_metadata is too low.

Comment (root-expert) [reply metadata.json:86]: Ah that's why itnsticked in endless loop.
Isn't it better to constraint it?

Also I expected the modulesync to bring the Gemfile up-to-date. Probably I missed something?

Comment (smortex) [reply metadata.json:86]: >  Isn't it better to constraint it?

Yes :-)  It was just FYI :-)

**Review by llowder — COMMENTED** [review]
I don't see any red flags, but the amount of change was pretty broad and beyond the scope of my knowledge, so I'll defer to someone else for a more in depth review and formal approval

**Pull request merged**